src/test_fixture/mod.rs

```diff
@@ -67,10 +70,10 @@ pub fn make_malicious_contexts<F: Field>(test_world: &TestWorld) -> [MaliciousCo
 /// # Panics
 /// Never, but then Rust doesn't know that; this is only needed because we don't have `each_ref()`.
 #[must_use]
-pub fn narrow_contexts<'a, F: Field, S: SecretSharing<F>>(
-    contexts: &[ProtocolContext<'a, S, F>; 3],
+pub fn narrow_contexts<C: Debug + ProtocolContext<F, Share = S>, F: Field, S: SecretSharing<F>>(
```

Comment:
`Debug` because there is a call to `unwrap` inside this function

Reply:
It's good that `ProtocolContext` is not `Debug` in the normal case.  It's OK to be `Debug` for tests, but we shouldn't need it for real code.